web/src/app/donate/DonateComponent.js

```diff
@@ -203,10 +201,17 @@ class DonateComponent extends Component {
                   <span className={styles.errorMessage}>{bountyErrorMessage}</span>
                 </Row>
                 <Row>
-                  <PayWithPayPalComponent/>
+                  <Button type="submit" bsStyle="success" onClick={this.submitForm} disabled={!this.isSubmitEnabled()}>Complete Payment with PayPal</Button>
                 </Row>
                 <Row>
-                  <Button type="submit" bsStyle="success" onClick={this.submitForm} disabled={!this.isSubmitEnabled()}>Submit</Button>
+                  {this.state.showPaymentOptions ?
```

Comment:
Lets flip this state back when there is an error so the user cannot submit an invalid form.

Reply:
Good catch. done.